src/components/FAQSection/QuestionFAQ/QuestionFAQ.styles.tsx

```diff
@@ -0,0 +1,62 @@
+import { Icon } from "@/components/Icon/Icon";
+import styled from "styled-components";
+
+export const SubSectionFAQ = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  padding-left: 0.5rem;
+  transition: 100ms;
+  margin-top: 0;
+  margin-left: 0.5rem;
+`;
+
+export const IconWrapper = styled(Icon)`
+  color: ${({ theme }) => theme.colors.black};
+  transition: 100ms linear;
+  flex-shrink: 0;
+`;
+
+export const QuestionContainer = styled.details`
+  display: flex;
+  width: 100%;
+  padding: 1rem 0;
+  margin: 1rem;
+  background-color: white;
+
+  box-shadow: 0px 0px 2px grey;
+
+  &:not([open]) ${IconWrapper} {
+    transform: rotate(45deg);
+  }
+
+  &[open] ${SubSectionFAQ} {
+    margin-top: 0.5rem;
+  }
+
```

Comment:
```suggestion
  &:not([open]) ${Summary}:hover {
    background: #cdcdcd20;
  }

```

Reply:
This way, the hover will only be displayed when the QuestionContainer is closed. i think it would be nice if it also appeared when the container is open, what do you think about this solution? 
```suggestion
  summary:hover {
    background: #cdcdcd20;
  }
```

Reply:
loved it